api/utilities/validators/user_validator.py

```diff
@@ -0,0 +1,23 @@
+from api.models.models import Users
+from api.middlewares.base_validator import ValidationError
+from api.utilities.messages.error_messages import SERIALIZATION_ERRORS
+
+
+def check_user(data):
+    """
+    checks if a user with similar email exists
+    """
+    user = Users.query.filter_by(email=data.get('email')).first()
+
+    if not user:
+        try:
+            user = Users(**data)
+            return user
+
+        except Exception as error:
```

Comment:
Catch specific errors and control how they are displayed to the user. It will not be very useful to pass on Python errors.